### PR TITLE
Yarn Berry: Ensure registry config is respected

### PR DESCRIPTION
**Pull request by jurre**
When a global/scoped registry is configured in the yarnrc, Dependabot should honor this for all requests, also for Yarn Berry, which prior to this change wasn't the case.

This ensures we correctly parse it etc.

### CODE REVIEW

File: npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/latest_version_finder.rb [hunk 1]
```diff
@@ -371,7 +371,8 @@ def registry_finder
             dependency: dependency,
             credentials: credentials,
             npmrc_file: npmrc_file,
-            yarnrc_file: yarnrc_file
+            yarnrc_file: yarnrc_file,
+            yarnrc_yml_fle: yarnrc_yml_file
```
Comment (jakecoffman) [inline npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/latest_version_finder.rb:375]:
```suggestion
            yarnrc_yml_file: yarnrc_yml_file
```

**Review by jakecoffman — APPROVED** [review]

**Comment by yarinsa** [comment]
+1 Thanks very helpful!

**Review by pavera — APPROVED** [review]

File: npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/latest_version_finder.rb (hunk 1 above)
Comment (S1983tt) [reply npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/latest_version_finder.rb]: ,🌠💲👍👍

**Comment by jurre** [comment]
> +1 Thanks very helpful!
> 
> How did you figure out the npmAuthIdent?
> 
> #5870

I don't think it's related? Let's debug your problem in the issue you've opened for it though 👍

**Pull request merged**